nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/dto/CandidateDto.java

```diff
@@ -19,7 +22,7 @@ public record CandidateDto(
     List<ApprovalDto> approvals,
     BigDecimal totalPoints,
     List<NoteDto> notes,
-    PeriodStatusDto periodStatus,
```

Comment:
I would keep `periodStatus` and mark it with deprecated and `TODO: Remove when frontend has migrated` to avoid deployment dependency.

Reply:
Evt. @JsonAlias

Reply:
Using following pattern to serialize period to "default" jsonMapping `period`, and `periodStatus`. I mean it is ugly to add duplicated property with another name to object:

```
    @Deprecated
    @JsonGetter("periodStatus")
    public PeriodStatusDto getPeriodStatus() {
        return this.period;
    }
```

Reply:
Test to remove:

```
    //TODO: To remove when frontend has been updated to use "period" property
    @Deprecated
    @Test
    void shouldDeserializeCandidateDtoContainingPeriodAndPeriodStatusProperties() throws JsonProcessingException {
        var candidate = CandidateDto.builder().withPeriod(PeriodStatusDto.builder().withId(randomUri()).build()).build();

        var node = JsonUtils.dtoObjectMapper.readTree(candidate.toJsonString());

        assertTrue(node.at("/period").isObject());
        assertTrue(node.at("/periodStatus").isObject());
    }
```